events/views.py

```diff
@@ -219,3 +225,20 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id):
```

Comment:
I think we should use here id and slug as we use for the detail page, to be more search engine friendly.